### DISCUSSION TITLE
Detonator is ticking at unexpected intervals

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#15164] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I noticed some strange behaviors with the detonator, the flashing light and ticking sound didn't match, so I looked into the code of detonator and found this:
```xml
<StatusEffect type="OnActive" target="This" interval="0.2" disabledeltatime="true">
  <sound file="Content/Items/WarningBeepSlow.ogg" range="750" frequencymultiplier="2.0" volume="0.75" />
</StatusEffect>
```
The code expects to hear a ticking sound every 0.2 seconds, but it's ticking very slowly.

In addition, after 2 seconds of activation, you hear a continuous detonating sound, but it doesn't detonate the explosive.

https://github.com/user-attachments/assets/56281a66-48e3-44bf-8fe9-16ca80a3d222

### Reproduction steps

1. activate a detonator

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
